### PR TITLE
Modify KAS Anchor flag for new spec

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
This PR applied new KAS anchor spec below.
- removed x-krn
- added x-chain-id flag

This PR added a error message into a log.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/sc/kas/anchor_test.go [hunk 1]
```diff
@@ -45,26 +45,26 @@ func testAnchorData() *types.AnchoringDataInternalType0 {
...
-func _TestExampleSendRequest(t *testing.T) {
-	url := "http://anchor-anchor-api.dev.klaytn.com/v1/anchor"
-	xkrn := "krn:1001:anchor:test:operator-pool:rp1"
-	user := ""
-	pwd := ""
+func TestExampleSendRequest(t *testing.T) {
+	url := "http://anchor-api.dev.klaytn.com/v1/anchor"
+	xChainId := "1001"
+	user := "Basic"
+	pwd := "S0FTSzVQREtBUEtYUUw0MEE3NzlSSkRPOnJHVlBVd3lXMjNVYkMzUEFORUxoQkJoaDgwYjFCQ2dBVWxwQW5KSlc"
```
Comment (aidan-kwon) [inline node/sc/kas/anchor_test.go:58]: Is that a real account of dev environment? The authentication information should not be revealed in public.

Comment (KimKyungup) [reply node/sc/kas/anchor_test.go:58]: Oops It's my mistakes. I will remote pwd.

Comment (aidan-kwon) [reply node/sc/kas/anchor_test.go]: Please, rebase your commit so that the pwd is not logged in history.

**Comment by KimKyungup** [comment]
@aidan-kwon I rebased and removed the comment to prevent the exposure of the key. Thanks.

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -638,9 +638,9 @@ var (
 		Name:  "kas.sc.anchor.operator",
 		Usage: "The operator address for KAS anchor",
 	}
-	KASServiceChainXKRNFlag = cli.StringFlag{
-		Name:  "kas.xkrn",
-		Usage: "The x-krn for KAS",
+	KASServiceChainXChainIdFlag = cli.StringFlag{
+		Name:  "kas.x-chain-id",
```
Comment (ehnuje) [inline cmd/utils/flags.go:642]: Just out of curiosity, what `x` stands for here?

**Review by ehnuje — APPROVED** [review]

File: node/sc/kas/anchor_test.go [hunk 3]
```diff
@@ -51,9 +51,9 @@ func testAnchorData() *types.AnchoringDataInternalType0 {
 	}
 }
 
-func _TestExampleSendRequest(t *testing.T) {
```
Comment (aidan-kwon) [inline node/sc/kas/anchor_test.go:54]: What do you want to test in this TC? It seems like a ping test for a dev server.

Comment (KimKyungup) [reply node/sc/kas/anchor_test.go:54]: This is an example test code for a dev server.
So I want to except it for common test.

File: cmd/utils/flags.go (hunk 2 above)
Comment (KimKyungup) [reply cmd/utils/flags.go:642]: KAS team @leejh903 said it is a general prefix for header of http request. 👍 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

Comment (KimKyungup) [reply cmd/utils/flags.go:642]: But the `x-` style is deprecated but KAS API use the convention.

File: node/sc/kas/anchor_test.go (hunk 3 above)
Comment (aidan-kwon) [reply node/sc/kas/anchor_test.go:54]: You enabled this test in this change.

Comment (KimKyungup) [reply node/sc/kas/anchor_test.go:54]: Oops...^^;

Comment (KimKyungup) [reply node/sc/kas/anchor_test.go:54]: Disabled. Thanks. :)

**Review by aidan-kwon — APPROVED** [review]

**Review by jeongkyun-oh — APPROVED** [review]

**Review by winnie-byun — APPROVED** [review]

**Review by ehnuje — APPROVED** [review]

**Pull request merged**